freqtrade/freqai/freqai_interface.py

```diff
@@ -760,12 +761,24 @@ def cache_corr_pairlist_dfs(self, dataframe: DataFrame, dk: FreqaiDataKitchen) -
                                "is included in the column names when you are creating features "
                                "in `populate_any_indicators()`.")
             self.get_corr_dataframes = not bool(self.corr_dataframes)
-        else:
+        elif self.corr_dataframes:
             dataframe = dk.attach_corr_pair_columns(
                 dataframe, self.corr_dataframes, dk.pair)
 
         return dataframe
 
+    def track_current_candle(self):
+        """
+        Checks if the latest candle appended by the datadrawer is
+        equivalent to the latest candle seen by FreqAI. If not, it
+        asks to refresh the cached corr_dfs, and resets the pair
+        counter.
+        """
+        if self.dd.current_candle > self.current_candle:
+            self.get_corr_dataframes = True
+            self.pair_it = 0
```

Comment:
Based on my tests, this disables the Inference output (`self.pair_it == self.total_pairs` is never reachable) - which is clearly not the desired outcome

Reply:
Fixed by changing from 0 to 1.